.appveyor.yml

```diff
@@ -25,6 +25,8 @@ cache:
   - '%AppData%\pip-cache'
 
 build_script:
+  # required to link py35 (x86 and x64)
+  - set PATH=%PATH%;C:\Program Files (x86)\Windows Kits\8.1\bin\x86
```

Comment:
This should have nothing to do with this [error](https://ci.appveyor.com/project/dwave-adtt/dwave-system/builds/35282055/job/8jpfxw1ffg64qr5j).

The path has to be fixed for setuptools to find the visual studio linker. And we're not compiling anything here.

Reply:
Unrelated, why are we hard-coding the SAPI token in appveyor conf?! @arcondello?

Reply:
Or, maybe a better question is: are these available on pull request build? (not sure how AppVeyor is configured here)

Reply:
Not decoded in PRs https://www.appveyor.com/docs/build-configuration/#secure-variables

Reply:
Yeah, I wasn't sure because I saw this option (unchecked) for dwave-system:

![image](https://user-images.githubusercontent.com/447371/93628428-66503100-f99b-11ea-8056-9b9ed00954f7.png)
It's not clear what's the baseline, when they say "only".

But it looks we're good.